package.json

```diff
@@ -10,7 +10,7 @@
     "url": "https://github.com/Sage/design-tokens"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=12"
```

Comment:
Why did you change the node version?

Reply:
I needed to run the library locally on node 12. I needed node12 common js module support. I'll recheck again if everything will be working on node 14.